packages/terra-infinite-list/src/InfiniteList.jsx

```diff
@@ -3,71 +3,58 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import 'terra-base/lib/baseStyles';
 import ResizeObserver from 'resize-observer-polyfill';
-import List from 'terra-list';
-import SelectableUtils from 'terra-list/lib/SelectableUtils';
+import List, {
+  Item, SectionHeader, SubsectionHeader,
+} from 'terra-list';
 import InfiniteUtils from './_InfiniteUtils';
 import styles from './InfiniteList.module.scss';
 
 const cx = classNames.bind(styles);
 
 const propTypes = {
   /**
-   * The child list items, of type InfiniteList.Item, to be placed within the infinite list.
-   * For further documentation of InfiniteList.Item see terra-list's ListItem.
+   * The child list items, of type InfiniteList Item, to be placed within the infinite list.
+   * For further documentation of InfiniteList Item see terra-list's ListItem.
    */
   children: PropTypes.node,
   /**
-   * Whether or not unselected items should be disabled.
-   * Helpful for enabling max row selection.
+   * Whether or not the list's child items should have a border color applied.
+   * One of `'none'`, `'standard'`, `'bottom-only'`.
    */
-  disableUnselectedItems: PropTypes.bool,
-  /**
-   * Whether or not the child list items has a disclosure indicator presented.
-   * The behavior is intended to be used with a single selection style list, as multi selection style list should not perform disclosures.
-   */
-  hasChevrons: PropTypes.bool,
+  dividerStyle: PropTypes.oneOf(['none', 'standard', 'bottom-only']),
   /**
    * An indicator to be displayed when no children are yet present.
    */
   initialLoadingIndicator: PropTypes.element,
-  /**
-   * Whether or not the child list items should have a border color applied.
-   */
-  isDivided: PropTypes.bool,
   /**
    * Determines whether or not the loading indicator is visible and if callbacks are triggered.
    */
   isFinishedLoading: PropTypes.bool,
-  /**
-   * Whether or not the list is selectable.
-   */
-  isSelectable: PropTypes.bool,
-  /**
-   * A callback event that will be triggered when selection state changes.
-   */
-  onChange: PropTypes.func,
   /**
    * Callback trigger when new list items are requested.
    */
   onRequestItems: PropTypes.func,
+  /**
+   * The padding styling to apply to the child list item content.
+   * One of `'none'`, `'standard'`, `'compact'`.
+   */
+  paddingStyle: PropTypes.oneOf(['none', 'standard', 'compact']),
   /**
    * An indicator to be displayed at the end of the current loaded children.
    */
   progressiveLoadingIndicator: PropTypes.element,
   /**
-   * An array of the currectly selected indexes.
+   * Accessibility role of the list, defaults to 'none'. If creating a list with selectable items, pass 'listbox'.
    */
-  selectedIndexes: PropTypes.array,
+  role: PropTypes.string,
```

Comment:
Should this be one of `none` or `listbox`?

Reply:
There are quite a few variants, so we can't really lock it down.